server/lib/flow-metrics.js

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var crypto = require('crypto');
+
+module.exports = function flowEventData(config, request) {
```

Comment:
@rfk in case it isn't obvious / you didn't realise, we have server tests under `tests/server`. You can run them with `npm run test-server`.